### ISSUE TITLE
Filename options

### ISSUE THREAD

**Issue by subhead** [ruial/9gag#9]
Is there a way to change the filenames of the saved images? Or is it planned to add such a feature?

**Comment by ruial** [comment]
Not really, I decided to use the same name they are using for the image, which contains the post ID.
Currently only the output folder name can be changed.
If you want to rename images you can use the file system module.
